ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/DecryptPioneerPayloads.java

```diff
@@ -118,13 +119,15 @@ private DecryptPioneerPayloads(ValueProvider<String> metadataLocation,
       JsonNode payload = json.get(FieldName.PAYLOAD);
 
       byte[] payloadData;
-      if (payload.get(SCHEMA_NAME).asText().equals(DELETION_REQUEST_SCHEMA_NAME)) {
-        // The deletion-request ping is special cased within the decryption
-        // pipeline. The Pioneer client requires a JWE payload to exist before
-        // it can be sent. The encrypted data is the empty string encoded with a
-        // throwaway key, in order to satisfy these requirements. We only need
-        // the envelope metadata to shred all client data in a study, so the
-        // encrypted data in the deletion-request ping is thrown away.
+      if (payload.get(SCHEMA_NAME).asText().equals(DELETION_REQUEST_SCHEMA_NAME)
+          || payload.get(SCHEMA_NAME).asText().equals(ENROLLMENT_SCHEMA_NAME)) {
```

Comment:
As you can see [here](https://phabricator.services.mozilla.com/D89718) the name of the schema is the same for both the "global consent" and the "study consent". The difference is the schema namespace and the fact that the study id is missing for "global consent". Do you need to account for that?

Reply:
Moreover, do I need to add a new schema [here](https://github.com/mozilla-services/mozilla-pipeline-schemas/tree/master/schemas/pioneer-debug)?

Reply:
It doesn't need to be accounted for here, but a new document namespace in mozilla-pipeline-schemas named `pioneer-meta` will need to be created with a `pioneer-meta/pioneer-enrollment/pioneer-enrollment.1.schema.json` schema where it is the empty schema. 

You can add the new schema to the debug study, too.

Reply:
@acmiyaguchi I created mozilla-services/mozilla-pipeline-schemas#611